spec/models/patron_spec.rb

```diff
@@ -49,7 +49,7 @@
 
   context 'with checkouts' do
     before do
-      fields[:circRecordList] = [{ key: 1, fields: {} }]
+      fields[:circRecordList] = [{ key: 1, fields: { status: 'ACTIVE' } }]
```

Comment:
I'm confused about this. The commit message says you were trying to fix active checkouts - but, no update to the specs themselves, just the stubbed item here... ?

Reply:
Which specs you expect to see changes? The only place I was able to easily filter out the inactive checkouts is in patron model checkouts method so I had to include status here.  Of course I might be missing sth. Maybe here there could be another spec for status Inactive to test if it is in the checkouts list or not but I kinda thought it would not matter much but I might be wrong.

Reply:
I get it now. Thanks.